From: LaserEyess <LaserEyess@users.noreply.github.com>
Date: Tue, 30 Dec 2025 16:27:24 -0500
Subject: Add ctrl+f shortcut for search

It was previously ctrl+k, so leave that as is. Multiple shortcuts can be
used to focus the filter box. Typeahead also still works in various
widgets.
---
 src/trg-main-window.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/trg-main-window.c b/src/trg-main-window.c
index 267bea0..02b3115 100644
--- a/src/trg-main-window.c
+++ b/src/trg-main-window.c
@@ -1888,7 +1888,8 @@ static gboolean window_key_press_handler(GtkWidget *widget, GdkEvent *event, gpo
 {
     TrgMainWindow *win = TRG_MAIN_WINDOW(widget);
 
-    if ((event->key.state & GDK_CONTROL_MASK) && event->key.keyval == GDK_k) {
+    if ((event->key.state & GDK_CONTROL_MASK)
+        && (event->key.keyval == GDK_k || event->key.keyval == GDK_f)) {
         gtk_widget_grab_focus(win->filterEntry);
         return TRUE;
     }
